sim/simx/cache_sim.cpp

```diff
@@ -673,8 +722,8 @@ CacheSim::CacheSim(const SimContext& ctx, const char* name, const Config& config
 	: SimObject<CacheSim>(ctx, name)
 	, CoreReqPorts(config.num_inputs, this)
 	, CoreRspPorts(config.num_inputs, this)
-	, MemReqPort(this)
-	, MemRspPort(this)
+	, MemReqPorts((1 << config.B), this)
```

Comment:
note that the cache memreqPorts should be sized to a new configuration parameter like num_mem_ports.